Source/OxyPlot/OxyPlot.Core.nuspec

```diff
@@ -15,10 +15,7 @@
   </metadata>
   <files>
     <file src="..\..\Output\PCL\OxyPlot.???" target="lib\portable-net45+netcore45+wpa81+wp8+MonoAndroid1+MonoTouch1+Xamarin.iOS10" />
-    <file src="..\..\Output\PCL\OxyPlot.???" target="lib\net45" />
-    <file src="..\..\Output\NET40\OxyPlot.???" target="lib\net40-client" />
-    <file src="..\..\Output\NET40\OxyPlot.???" target="lib\net40" />
-
+    <file src="..\..\Output\PCL\OxyPlot.???" target="lib\net45" />    
```

Comment:
This line looks a bit weird. Do you really want a PCL inside a net45 lib? If you don't include it, then it should auto-fallback on the PL anyway.